hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/filedrops/FileDropGenerater.java

```diff
@@ -37,8 +37,8 @@
 	 */
 	protected String buildFileNameParameters(Exchange exchange, String transactionid) {		
 		String accessToken = (String) exchange.getIn().getHeader(Util.AUTHORIZATION);
-		String msgType = (String)exchange.getIn().getHeader(Util.HEADER_MESSAGE_TYPE);
-		String sendingFacility = (String)exchange.getIn().getHeader(Util.HEADER_SENDING_FACILITY);
+		String msgType = (String)exchange.getIn().getHeader(Util.PROPERTY_MESSAGE_TYPE);
```

Comment:
I think this should be exchange.getProperty()

Reply:
This was still working because there is additional code to handle filedrops before the properties are set. Fixed it to avoid having to get the value again.